src/main/java/com/myworkspace/reportapp/entity/customer/Manager.java

```diff
@@ -0,0 +1,56 @@
+package com.myworkspace.reportapp.entity.customer;
+
+import jakarta.persistence.DiscriminatorValue;
+import jakarta.persistence.Entity;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+import java.util.Objects;
+
+
+@Entity
+@DiscriminatorValue("MANAGER")
+@NoArgsConstructor
+@Getter
+public class Manager extends User {
+
+    private String firstName;
+    private String lastName;
+    private String password;
```

Comment:
Storing passwords in plain text is generally not recommended for security reasons. It's a better practice to hash and salt passwords before storing them in a database. You could consider using libraries like BCrypt to securely hash passwords.

Reply:
Once more time, I create hash with salt passwords use BCrypt in service package. In entity I don't want add many others method, because Entity classes should use also SOLID metods. In this case S :)